src/blocks/server.blocks/middleware/__proxy-example/middleware__proxy-example.node.js

```diff
@@ -45,6 +45,20 @@ modules.define('middleware__proxy-example', ['config', 'constants', 'logger', 'u
                                 model.putToCache(sha(url), html);
                                 res.end(html);
                             });
+                        } else if (/showcase\.html$/.test(url)) {
+                            var urlArr = url.split('/'),
+                                lib = urlArr[0],
+                                version = urlArr[1];
+
+                            // href="../../common.blocks/menu/menu.ru.md" -> href="lib/version/desktop/menu"
+                            response = response.replace(/href="([^"]*)"/g, function (str) {
+                                if (str.indexOf('showcase') > -1) return str; // ignore _showcase.css|js
+                                return 'target="_top" href="' + '/libs/' + lib + '/' + version + '/desktop' +
```

Comment:
I am agree with @tormozz48 
In addition, it would be good to add a comment to
 `str.match(/\/(\w+|\w+-\w+)\.(ru|en)\.md(#.+[^"])?/(g)[0].replace(/\.(ru|en)\.md/, '/') + '"';`
for example: `source: ../../common.blocks/menu/menu.ru.md` -> /menu.ru.md replace to -> /menu/